Makefile

```diff
@@ -79,4 +79,7 @@ clean:
 
 .PHONY: release
 release:
+	SHELL:=/bin/bash
```

Comment:
What does this line do?

Reply:
Sets the shell to be bash for the release target, and only the release target. Might not be a good idea for portability reasons (?) but I wanted to make sure that I could use sed in the Makefile. It might be unneeded.

Reply:
This isn't necessary. Our release process always runs in the the `quay.io/fossas/fossa-cli` Docker container, which uses `bash`.